eng/pipelines/templates/steps/generate-doc.yml

```diff
@@ -0,0 +1,17 @@
+steps:
+  - script: |
+      npm i -g typedoc typescript
```

Comment:
Should we pin typedoc to 0.17.x so that we don't break the next time they make a major version bump?

Reply:
yeah we can do that, shoudl i pint typedoc also in that case?

Reply:
I'd say, pin both typedoc and typescript!

Reply:
pinning typedoc to 0.16.x